plugins/modules/aci_l4l7_concrete_interface.py

```diff
@@ -0,0 +1,355 @@
+short_description: Manage L4-L7 Concrete Interfaces (vns:CIf)
+description:
+- Manage L4-L7 Concrete Interfaces.
+options:
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  device:
+    description:
+    - The name of an existing logical device.
+    type: str
+  concrete_device:
+    description:
+    - The name of an existing concrete device.
+    type: str
+  name:
+    description:
+    - The name of the concrete interface.
+    type: str
+    aliases: [ concrete_interface ]
+  pod_id:
+    description:
+    - The pod ID the concrete interface exists on.
+    type: int
+  node_id:
+    description:
+    - The node ID the concrete interface exists on.
+    - For Ports and Port-channels this is a single node-id.
+    - For vPCs this is a hyphen separated pair of node-ids, e.g. "201-202".
+    type: str
+  path_ep:
+    description:
+    - The path to the physical interface.
+    - For single ports, this is the port name, e.g. "eth1/15".
+    - For Port-channels and vPCs, this is the Interface Policy Group name.
+    type: str
+  vnic_name:
+    description:
+    - The concrete interface vNIC name.
+    type: str
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+
+seealso:
+- module: aci_l4l7_device
+- module: aci_l4l7_concrete_device
+- name: APIC Management Information Model reference
+  description: More information about the internal APIC class B(vns:CIf)
+  link: https://developer.cisco.com/docs/apic-mim-ref/
+author:
+- Tim Cragg (@timcragg)
+"""
+
+EXAMPLES = r"""
+- name: Add a new concrete interface on a single port
+  cisco.aci.aci_l4l7_concrete_interface:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    tenant: my_tenant
+    device: my_device
+    concrete_device: my_concrete_device
+    name: my_concrete_interface
+    pod_id: 1
+    node_id: 201
+    path_ep: eth1/16
+    state: present
+  delegate_to: localhost
+
+- name: Add a new concrete interface on a vPC
+  cisco.aci.aci_l4l7_concrete_interface:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    tenant: my_tenant
+    device: my_device
+    concrete_device: my_concrete_device
+    name: my_concrete_interface
+    pod_id: 1
+    node_id: 201-202
+    path_ep: my_vpc_ipg
+    state: present
+  delegate_to: localhost
+
+- name: Delete a concrete interface
+  cisco.aci.aci_l4l7_concrete_interface:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    tenant: my_tenant
+    device: my_device
+    concrete_device: my_concrete_device
+    name: my_concrete_interface
+    pod_id: 1
+    node_id: 201-202
+    path_ep: my_vpc_ipg
+    state: absent
+  delegate_to: localhost
```

Comment:
absent has only 4 required parameters, if the rest is not required, we can remove them from examples ["state", "absent", ["tenant", "device", "concrete_device", "name"]]

Reply:
Updated